src/main/java/com/univocity/parsers/containers/Column.java

```diff
@@ -0,0 +1,10 @@
+package com.univocity.parsers.containers;
+
+/**
+ * @author naveen.kasthuri
+ * A column reference. Only enums should implement this interface.
+ * Column header must match enum such that columnHeaderEnum.toString().equals(columnHeader) is true.
+ * @see com.univocity.parsers.examples's RowObjectExample for an implementation example
+ */
+public interface Column {
```

Comment:
Consider extending the Column interface to something like `ColumnIndex` enumerations that provide the index of a column as well - so it can work with files that don't have headers.